Overview.src.html

```diff
@@ -1874,39 +1929,11 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <span>process response</span> for <var>response</var>.
 
  <li>
-  <p>If <var>internalResponse</var>'s
-  <span title=concept-response-body>body</span> is null, run these substeps:
-
-  <ol>
-   <li><span>Queue a fetch task</span> on <var>request</var> to
-   <span>process response body</span> for <var>response</var>.
-
-   <li><p><span>Queue a fetch-done task</span> using <var>request</var> and
-   <var>response</var>.
-  </ol>
-
- <li>
-  <p>Otherwise, if <var>internalResponse</var>'s
-  <span title=concept-response-body>body</span> is non-null, run these substeps:
-
+  <p>Run these substeps <span data-anolis-spec=html>in parallel</span>:
```

Comment:
At this point we are already running in parallel so I'm not sure this is needed.